myapp/src/components/Thread/Form/FormElements.js

```diff
@@ -0,0 +1,6 @@
+import styled from 'styled-components';
+
+export const CommentInput = styled.input`
```

Comment:
Since this is the only thing being exported, you'll have to remove `export` from line 3 and add `export default CommentInput` at the bottom. ESLint will throw an error if you don't and then you'll have to change how it's being imported on other files that import `CommentInput`